ckanext/depositar_theme/fanstatic/scss/_masthead.scss

```
@@ -0,0 +1,240 @@
+
+        a {
+          color: $accountMastheadListLinkTextColor;
+          padding: 0 10px;
+          font-size: 14px;
+          display: block;
+          line-height: 21px;
+
+          &:hover {
+            background-color: $accountMastheadListLinkHoverBackgroundColor;
+            text-decoration: none;
+
+            .badge {
+              color: $accountMastheadListLinkBadgeHoverTextColor;
+              background-color: $accountMastheadListLinkBadgeHoverBackgroundColor;
+            }
+          }
+
+          .text {
+            display: none;
+          }
+
+          .user-image {
+            border-radius: 4px;
+          }
+
+          .badge {
+            color: $accountMastheadListLinkBadgeTextColor;
+            background-color: $accountMastheadListLinkBadgeBackgroundColor;
+            margin-left: 3px;
+            padding: 1px 6px;
+            font-size: 12px;
+          }
+        }
+      }
+    }
+  }
+}
+
+.masthead {
+  background-color: $mastheadBackgroundColor;
+  background-image: none;
+
+  &.navbar {
+    padding: 5px 0;
+  }
+
+  .navbar-collapse {
+    padding-top: 0.5rem;
+    justify-content: flex-end;
+  }
+
+  .logo img {
+    width: 185px;
+    padding-left: 5px;
+  }
+
+  .navigation {
+    margin-top: 5px;
+
+    .nav-pills {
+      li {
+        a {
+          display: block;
+          padding: 10px 15px;
+          border-radius: 0;
+          line-height: 7px;
+          text-decoration: none;
+
+          &::after {
+            content: '';
+            display: block;
+            width: 0;
+            margin-top: 6px;
+            height: 3px;
+            background-color: $mastheadListLinkAfterBackgroundColor;
+            -webkit-transition: all 0.3s cubic-bezier(0.175, 0.885, 0.32, 1.02);
+            transition: all 0.3s cubic-bezier(0.175, 0.885, 0.32, 1.02);
+            -webkit-transition-property: width, color;
+            transition-property: width, color;
+          }
+
+          &:hover {
+            background-color: $mastheadBackgroundColor;
+
+            &::after {
+              width: 100%;
+            }
+          }
+        }
+
+        &.active {
+          a {
+            background-color: $mastheadBackgroundColor;
+
+            &::after {
+              width: 100%;
+              margin-top: 6px;
+              height: 3px;
+              background-color: $mastheadListLinkAfterBackgroundColor;
+              border-radius: 0;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .nav > li > a,
+  .nav > li > a:focus,
+  .nav > li > a:hover,
+  .nav > .active > a,
+  .nav > .active > a:hover,
+  .nav > .active > a:focus {
+    color: $mastheadTextColor;
+    border-bottom: none;
+    font-size: 15px;
+  }
+
+  .btn-navbar,
+  .btn-navbar:hover,
+  .btn-navbar:focus {
+    background-color: $mastheadBackgroundColor;
+    margin-top: 0;
+    color: $mastheadTextColor;
+    font-size: 25px;
+    border: none !important;
+    outline: unset;
+  }
+
```

避免 debug = True 時導致功能列右偏

```suggestion
  .debug {
    position: absolute;
    top: 37px;
    left: 10px;
    color: rgba(255, 255, 255, 0.5);
  }
```